level1c4pps/gac2pps_lib.py

```diff
@@ -87,7 +87,7 @@ def update_ancilliary_datasets(scene):
     first_jan_1970 = np.array([datetime(1970, 1, 1, 0, 0, 0)]).astype('datetime64[ns]')
     scanline_timestamps = np.array(scene['qual_flags'].coords['acq_time'] -
                                    first_jan_1970).astype(dtype='timedelta64[ms]').astype(np.float64)
-    scene['scanline_timestamps'] = xr.DataArray(da.from_array(scanline_timestamps),
+    scene['scanline_timestamps'] = xr.DataArray(da.from_array(scanline_timestamps, chunks=1024),
```

Comment:
In more recent dask versions you have to specify a chunksize. I chose 1024, then you have roughly 12 chunks per file.

Reply:
Good to know that chunksizes will have to be specified!